client/public/js/actions/EventDetailActions.js

```diff
@@ -13,6 +13,13 @@ var EventDetailActions = {
       name: event.name,
       location: event.location
     });
+  },
+
+  create: function(newObservation){

```

Comment:
Probably would be better to name this something like `createObservation`. Since it's in the EventDetailActions file, `create` sounds like it would create a new event.